xml/containers_basics.xml

```diff
@@ -1,78 +1,202 @@
- </para>
- <itemizedlist mark="bullet" spacing="normal">
-  <title>Advantages of Using Containers</title>
-  <listitem>
-   <para>
-    Isolate applications through self-contained units.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    Provide nearly native performance, as containers manage allocation of
-    resources in real time.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    Control network interfaces and apply resources inside containers through
-    <xref linkend="gloss-vt-lxc-cgroups"/>.
-   </para>
-  </listitem>
- </itemizedlist>
- <itemizedlist mark="bullet" spacing="normal">
-  <title>Limitations of Containers</title>
-  <listitem>
-   <para>
-    Containers run inside the host system's kernel, thus they cannot use
-    different kernels or different kernel versions.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    Only Linux-based applications can be containerized.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    Containers are not secure, and the overall security depends on the host
-    system. Containerized applications can be secured through &aa; or &selnx;
-    profile.
-   </para>
-  </listitem>
- </itemizedlist>
+  <title>Introduction to Linux Containers</title>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker/>
+    </dm:docmanager>
+  </info>
+  <para>
+    Linux containers offer a lightweight virtualization method to run multiple
+    virtual environments (containers) simultaneously on a single host. Unlike
+    technologies like &xen; or &kvm;, where the processor simulates a complete
+    hardware environment and controls virtual machines, containers provide
+    virtualization at the operating system level, where the kernel controls the
+    isolated containers.
+  </para>
+  <itemizedlist mark="bullet" spacing="normal">
+    <title>Advantages of Using Containers</title>
+    <listitem>
+      <para>
+        Containers make it possible to solate applications through self-contained units.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Containers provide nearly native performance, as containers manage
+        allocation of resources in real time.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        It is possible to control network interfaces and apply resources inside
+        containers through <xref linkend="gloss-vt-lxc-cgroups"/>.
+      </para>
+    </listitem>
+  </itemizedlist>
+  <itemizedlist mark="bullet" spacing="normal">
+    <title>Limitations of Containers</title>
+    <listitem>
+      <para>
+        Containers run inside the host system's kernel, and they cannot use
+        different kernels or different kernel versions.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Only Linux-based applications can be containerized.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Containers are not secure, and the overall security depends on the host
```

Comment:
They can be made secure (and most users assume they are), but it's harder to achieve as with VM due to the bigger attack surface.